presto-main/src/main/java/com/facebook/presto/testing/InterfaceTestUtils.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.testing;
+
+import java.lang.reflect.Method;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.String.format;
+
+public final class InterfaceTestUtils
+{
+    private InterfaceTestUtils() {}
+
+    // Note: this should be in sync with com.facebook.presto.plugin.base.testing.InterfaceTestUtils#checkEverythingImplemented
```

Comment:
This mentioned class does not seem to exist. I'm guessing this is no longer relevant?

Reply:
Good catch. It was removed in the second fixup. Needs update.